OpenRA.Game/Map/MapCache.cs

```diff
@@ -41,11 +41,8 @@ public sealed class MapCache : IEnumerable<MapPreview>, IDisposable
 
 		readonly List<MapDirectoryTracker> mapDirectoryTrackers = new List<MapDirectoryTracker>();
 
-		/// <summary>
-		/// If a map was added oldUID will be null, if updated oldUId will point to the outdated map
-		/// Event is not called when map is deleted
-		/// </summary>
-		public event Action<string, string> MapUpdated = (oldUID, newUID) => { };
+		readonly Dictionary<string, string> mapUpdates = new Dictionary<string, string>();
+		public string LastUpdatedMap = null;
```

Comment:
Are we able to make this `public string LastUpdatedMap { get; private set; }` to ensure that others can't modify it?

Reply:
sure